electron/src/main.ts

```diff
@@ -328,21 +331,22 @@ const renameLogFile = () => {
 };
 
 class ElectronWrapperInit {
-  debug: debug.IDebugger;
+  logger: logdown.Logger;
 
   constructor() {
-    this.debug = debug('ElectronWrapperInit');
+    this.logger = logdown('wire-desktop/main.ts', {
+      logger: console,
+      markdown: false,
+    });
```

Comment:
Why not use the logger from line 32?

Reply:
because it's a different class logging

Reply:
But it has the same name in the logger..?

Reply:
yes, that should be changed